deps/spidershim/src/accessor.h

```diff
@@ -254,35 +284,20 @@ bool SetAccessor(JSContext* cx,
   // TODO: What should happen with "settings"?  See
   // https://github.com/mozilla/spidernode/issues/141
 
-  JS::RootedObject getterObj(cx, CreateAccessor(cx, getter, name, data, signature));
+  JSObject* getterObj = CreateAccessor(cx, getter, name, data, signature);
```

Comment:
No, this change is wrong.  getterObj is now unrooted across the call to CreateAccessor for the setter, which can GC.  This needs to stay rooted.